frontend/src/pages/modelRegistry/screens/utils.ts

```diff
@@ -99,22 +100,25 @@ export const getCustomPropString = <
   return '';
 };
 
-export const getCatalogModelDetailsProps = (
-  customProps: ModelRegistryCustomProperties,
-): CatalogModelDetailsParams => ({
-  sourceName: getCustomPropString(customProps, '_registeredFromCatalogSourceName'),
-  repositoryName: getCustomPropString(customProps, '_registeredFromCatalogRepositoryName'),
-  modelName: getCustomPropString(customProps, '_registeredFromCatalogModelName'),
-  tag: getCustomPropString(customProps, '_registeredFromCatalogTag'),
-});
-
-export const getPipelineModelCustomProps = (
-  customProps: ModelRegistryCustomProperties,
-): PipelineModelCustomProps => ({
-  project: getCustomPropString(customProps, '_registeredFromPipelineProject'),
-  runId: getCustomPropString(customProps, '_registeredFromPipelineRunId'),
-  runName: getCustomPropString(customProps, '_registeredFromPipelineRunName'),
-});
+export const getCatalogModelDetailsProps = (artifact: ModelArtifact): CatalogModelDetailsParams => {
+  // Check for modelSource* properties using the new standardized approach
+  if (artifact.modelSourceKind === ModelSourceKind.CATALOG && artifact.modelSourceName) {
+    return {
+      sourceName: artifact.modelSourceClass || '',
+      repositoryName: artifact.modelSourceGroup || '',
+      modelName: artifact.modelSourceName,
+      tag: artifact.modelSourceId || '',
+    };
+  }
+
+  // Return empty values if no valid catalog information is available
+  return {
+    sourceName: '',
+    repositoryName: '',
+    modelName: '',
+    tag: '',
+  };
+};
```

Comment:
Hmmm.... this function is redundant with your new `modelSourcePropertiesToCatalogParams` and no longer used except in your tests. I think you can remove it